x-pack/plugins/session_view/public/components/session_view/styles.ts

```diff
@@ -26,9 +26,22 @@ export const useStyles = ({ height = 500 }: StylesDeps) => {
       height: `${height}px`,
     };
 
+    const searchBar: CSSObject = {
+      position: 'relative',
+      marginLeft: `${euiTheme.size.xs} !important`, // 
```

Comment:
are you sure all these `!important` are needed? we should use `!important` only as last resource